### PR TITLE
✅  🚀  speed up test-amp-video-analytics test

### PR DESCRIPTION
**Pull request by rileyajones**
The CircleCI insights dashboard has flagged `amp-video with video analytics chrome Standalone environment control buttons should manipulate video box behavior` as one of the slowest e2e tests.
[Insights - ampproject_amphtml_workflows_CircleCI.pdf](https://github.com/ampproject/amphtml/files/6361644/Insights.-.ampproject_amphtml_workflows_CircleCI.pdf)

This seems to be due to the two synchronous 5000ms awaits. To address this I have changed the player to use a shorter video to ensure a smaller period of time is equivalent to a similar play percentage and decreased the await times.

### Before
image

### After
image

### CODE REVIEW

File: extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ describes.endtoend(
       await expect(isPaused(videoElem1)).to.be.false;
       await expect(isPaused(videoElem2)).to.be.true;
 
-      // Sleep 5 seconds for the `video-percentage-played` event trigger
+      // Sleep 1 second for the `video-percentage-played` event trigger
```
Comment (rileyajones) [inline extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js:49]: @micajuine-ho is it necessary that these be synchronous? Or could we play both videos at the same time and get the same behavior with only a single call to `sleep`?

**Review by rsimha — APPROVED** [review]

File: extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js (hunk 1 above)
Comment (micajuine-ho) [reply extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js:49]: I think it's best if they're sync, RequestBank (that captures requests) has been finicky in the past so keeping it sync should help prevent potential future flakes.

**Review by micajuine-ho — APPROVED** [review]
Thanks for these improvements!

File: extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js (hunk 1 above)
Comment (rileyajones) [reply extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js:49]: Alright, that makes sense but it seemed worth checking.

**Pull request merged**